build/builder.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -eu
+
+image="cockroachdb/builder"
+
+function init() {
```

Comment:
Yeah, but I'd rather leave this as intact as possible to make it easier to propagate changes.
right now, the only diff is in the paths.